modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeveloperKeys.testcase

```diff
@@ -96,7 +96,7 @@ definition {
 			CP2NavigationMenu.clickOnProductButton(product = "${testProduct}");
 		}
 
-		task ("Open the dropdown and verify that there are 5 DXP versions") {
+		task ("Open the dropdown and assert if have 5 versions") {
```

Comment:
I think the "Open the dropdown and verify that there are 5 DXP versions" is better written than the current one. Would you please return it to "Open the dropdown and verify that there are 5 DXP versions" @styller1 ?

Reply:
Alright